lib/agent.js

```diff
@@ -0,0 +1,15 @@
+const agent = require('auth0-instrumentation');
+const config = require('../conf');
+const EventLogger = require('auth0-common-logging').EventLogger;
+const package = require('../package.json');
+
+const env = config.toEnv();
+
+agent.init(package, env, null, { fileRotationSignal: 'SIGHUP' });
+
+agent.eventLogger  = new EventLogger(agent.logger);
+agent.eventLogger.watch(process);
+
+agent.metrics.startResourceCollection();
```

Comment:
We could get event loop red herrings if it starts monitoring and the db is being loaded into memory. Unless that's the intended behavior.

Reply:
That should not longer happen because we are spawning a new process to check the database consistency and repair